nixos/modules/virtualisation/nixos-containers.nix

```diff
@@ -157,9 +159,11 @@ let
       # Kill signal handling means systemd-nspawn will pass a system-halt signal
       # to the container systemd when it receives SIGTERM for container shutdown;
       # containerInit and stage2 have to handle this as well.
+      # TODO: fix shellcheck issue properly
+      # shellcheck disable=SC2086
```

Comment:
Yeah this is the fun part isn't it.

Reply:
Yeah, I still want to fix it properly, but that will take more effort.